opencog/atomspace/atom_types.script

```diff
@@ -76,6 +76,9 @@ EXISTS_LINK <- BIND_LINK
 AVERAGE_LINK <- FORALL_LINK
 SATISFYING_SET_LINK <- BIND_LINK
 
+// Function constructor (predicate and schema)
+LAMBDA_LINK <- BIND_LINK
```

Comment:
OK, Linas, there was indeed a confusion on my part regarding the semantics of the BindLink, basically I didn't imaging that the implicationlink was mandatory. So now I understand the binding exaclt connecting P and Q.

Yes, it feels to me too that the use of the ImplicationLink is not right, I'll look carefully at issue #1445